sixtyfps_runtime/rendering_backends/gl/glwindow.rs

```diff
@@ -322,16 +323,17 @@ impl PlatformWindow for GLWindow {
         if w <= 0. || h <= 0. {
             if let Some(component_rc) = self.self_weak.upgrade().unwrap().try_component() {
                 let component = ComponentRc::borrow_pin(&component_rc);
+                let hor_info = component.as_ref().layout_info(Orientation::Horizontal);
                 if w <= 0. {
-                    let info = component.as_ref().layout_info(Orientation::Horizontal);
-                    w = info.preferred_bounded();
+                    w = hor_info.preferred_bounded();
                     must_resize = true;
                 }
+                let ver_info = component.as_ref().layout_info(Orientation::Vertical);
                 if h <= 0. {
-                    let info = component.as_ref().layout_info(Orientation::Vertical);
-                    h = info.preferred_bounded();
+                    h = ver_info.preferred_bounded();
                     must_resize = true;
                 }
+                resizable = Some(hor_info.min != hor_info.max && ver_info.min != ver_info.max);
```

Comment:
```suggestion
                resizable = Some(hor_info.min < hor_info.max && ver_info.min < ver_info.max);
```

I believe using `<` instead of `!=` will silence clippy warning about comparing float.
Alternatively one could also round it to integer.

Reply:
That's a good point and I find your suggestion slightly more readable :)